plugins/woocommerce/package.json

```diff
@@ -688,7 +688,6 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
-				"node_modules/@woocommerce/classic-assets/build",
```

Comment:
I think we still need to include the `node_modules/@woocommerce/classic-assets/build` here.

I was also encountering the similar thing, I found that if you run `pnpm install` in sub directory of the project, it may update `dependencyOutputs` in wrong way. If you run `pnpm install` in the root directory of the project, it will not change `dependencyOutputs`.